content/en/events/isc-2021.md

```diff
@@ -721,4 +721,4 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 All related conversations in the InnerSource Slack Channel will still be considered to fall under [Chatham House Rules](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed can be shared but not attributed.
 
-Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```

Comment:
```suggestion
Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

Reply:
Change made suggested by @yuhattor helps address https://github.com/InnerSourceCommons/innersourcecommons.org/issues/454 and looks good to me - but not sure if clicking "commit suggestion" does that to Cristina's branch or the main website - so will leave someone else to do that. Thanks!

Reply:
Thanks so much @clcoffey for making these changes.